msdsn/conn_str.go

```diff
@@ -223,7 +222,7 @@ func Parse(dsn string) (Config, map[string]string, error) {
 			var err error
 			p.KrbCache, err = setupKerbCache(krbCache)
 			if err != nil {
-				return p, params, fmt.Errorf("cannot read kerberos cache file: %v", err)
+				return p, params, fmt.Errorf("cannot read kerberos cache file: %v", err.Error())
```

Comment:
This should be:
`return p, params, fmt.Errorf("cannot read kerberos cache file: %w", err)`